src/projects/automation/workflows/connectors.md

```diff
@@ -0,0 +1,27 @@
+# Connectors
+
+Connectors allow you to run the workflow via a HTTP Request, therefore allowing other software to connect to your workflow.
+
+There are two parts to a connector, the **transformer** which takes in the request and the **generator** which generates the response.
```

Comment:
We don't refer to transformers and generators as transformers or generators in the UI, they're referred to as `handler` and `response`

Reply:
@chrismilway is this maybe a time to make this consistent then? In the TDK they are referred to as Transformer and Generator?

Reply:
No, the UI says handler and response

Reply:
Darn